fetch.bs

```diff
@@ -1825,14 +1825,14 @@ run these steps:
 </ol>
 
 
-<h3 id=client-hints-list>Client hints list</h3>
+<h3 id=client-hints-list>Client-hints set</h3>
 
-<p class=note>This section will be integrated into HTTP Client Hints.
+<p class=note>This section will be integrated into HTTP Client-Hints.
```

Comment:
Is that still the plan? In any event, this section should have a `<p class=XXX>` explaining that currently the integration is not well defined.